components/common/footer.js

```diff
@@ -1,19 +1,49 @@
 import React from 'react'
+import { useAuth } from '../../utils/useAuth'
+
 import { Flex, List, ListItem, Text, Box } from '@chakra-ui/core'
 
 import TextLink from './textLink'
 import styles from './footer.module.scss'
 
+const loggedOutLinks = [
```

Comment:
since i added a constants file we can start moving these there

Reply:
In this case if we're only using them in the one component (footer), doesn't it make more sense to just keep them in the only file they're used in?

Reply:
yea you're probably right :)